File: scripts/fix_cover_dua_kuasa.py
import requests

API_BASE = 'https://api.shortlovers.id/api'
ADMIN_KEY = '00ca04e3e2702be565d7bf44e783255247708289bce9b2fb6187a2e117f87fd14'
HEADERS = {'x-admin-key': ADMIN_KEY, 'Content-Type': 'application/json'}

# ID Drama dari output terminal kamu sebelumnya
DRAMA_ID = 'w6pwucz8bgm8nv9lf8uv52xv'

# Cover R2 yang benar
NEW_COVER = 'https://stream.shortlovers.id/netshortv2/dua-kuasa-menjadi-satu/cover.webp'

print(f"Memperbaiki cover untuk drama ID: {DRAMA_ID}")
print(f"Cover baru: {NEW_COVER}")

try:
    url = f"{API_BASE}/admin/dramas/{DRAMA_ID}"
    r = requests.patch(url, headers=HEADERS, json={'cover': NEW_COVER})
    
    if r.status_code == 200:
        print("BERHASIL! Cover sudah diperbarui menggunakan API.")
        print("Silakan restart aplikasi mobile kamu.")
    else:
        print(f"GAGAL: {r.status_code} - {r.text}")
except Exception as e:
    print(f"Error: {e}")

File: scripts/fix_cover_dua_kuasa.mjs
import pg from 'pg';

const { Client } = pg;
const client = new Client({
  connectionString: 'postgresql://supabase_admin:GoZViiH1AXLl73BqLdKDtpeGgwUzfW64@141.11.160.187:5432/postgres'
});

async function run() {
  try {
    console.log("Menghubungkan ke database...");
    await client.connect();
    
    const slug = 'dua-kuasa-menjadi-satu';
    const r2_cover = `https://stream.shortlovers.id/netshortv2/${slug}/cover.webp`;
    
    const res = await client.query("UPDATE dramas SET cover = $1 WHERE title ILIKE '%Dua Kuasa Menjadi Satu%' RETURNING id, title, cover", [r2_cover]);
    
    if (res.rows.length > 0) {
      console.log("BERHASIL memperbaiki cover URL!");
      console.log("Drama:", res.rows[0].title);
      console.log("New Cover:", res.rows[0].cover);
    } else {
      console.log("Drama tidak ditemukan di DB.");
    }
  } catch (e) {
    console.error("DB Error:", e);
  } finally {
    await client.end();
  }
}

run();
